r/R/csv.R

```diff
@@ -300,6 +300,11 @@ CsvTableReader$create <- function(file,
                                   convert_options = CsvConvertOptions$create(),
                                   ...) {
   assert_is(file, "InputStream")
+
+  if (!identical(read_options$encoding, "UTF-8")) {
```

Comment:
Do we want to handle the various spellings of "utf-8" here? (lowercase, uppercase, with or without a hyphen)